test/tests/rackhd20/test_rackhd20_api_workflows.py

```diff
@@ -69,5 +70,143 @@ def test_api_20_workflows_put_ipmi(self):
         api_data = fit_common.rackhdapi("/api/2.0/workflows/graphs/" + 'Graph.Obm.Ipmi.CreateSettings.Test')
         self.assertEqual(api_data['status'], 200, 'Incorrect HTTP return code, expected 200, got:' + str(api_data['status']))
 
+    def test_api_20_workflows_waitOn_anyOf(self):
```

Comment:
Can you please add comments as to what you are testing here? 
There isn't an overall comment in the PR nor in the script. Not sure what this test is testing or verifying.  Thanks.